build/lib/locFunc.js

```diff
@@ -344,7 +332,7 @@ exports.refreshLangpacks = refreshLangpacks;
  * Function for adding replacing ads language packs with vscode ones.
  * For new languages, remember to add to i18n.extraLanguages so that it will be recognized by ADS.
 */
-function renameVscodeLangpacks() {
+async function renameVscodeLangpacks() {
```

Comment:
You're not await'ing anything in this function that I can tell which means this is unnecessary currently. All the code you added is synchronous